### ISSUE TITLE
Control-o opens both Snap!'s open project dialog and a file open dialog in Chrome

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2734]
Chrome has this shortcut

> Open a file from your computer in Chrome | Ctrl + o + Select a file

At least in Windows 10.

**Comment by jmoenig** [comment]
yep, that's a new behavior. On a Mac it'll work if you use the ctrl key instead of the cmd key. But we need to suppress the browser from taking over cmd/ctrl+o. Thanks!
